.travis.yml

```diff
@@ -29,6 +29,7 @@ install:
   - pip install unidecode
   - pip install suds
   - pip install http://xmlsoft.org/sources/python/libxml2-python-2.6.21.tar.gz
+  - pip install https://github.com/odoo-brazil/pybrasil/archive/master.zip
```

Comment:
Please put all these in a `requirements.txt` file. No reason to invoke `pip` multiple times here.